intyg-proxy-service/integration-pu-v5/src/main/java/se/inera/intyg/intygproxyservice/integration/pu/v5/client/PuClientV5.java

```diff
@@ -0,0 +1,104 @@
+package se.inera.intyg.intygproxyservice.integration.pu.v5.client;
+
+import static se.inera.intyg.intygproxyservice.integration.api.constants.PuConstants.PU_PROFILE_V5;
+import static se.inera.intyg.intygproxyservice.integration.pu.v5.configuration.configuration.PuConstants.CODE_COORDINATION_NUMBER;
+import static se.inera.intyg.intygproxyservice.integration.pu.v5.configuration.configuration.PuConstants.CODE_PERSONAL_ID;
+import static se.inera.intyg.intygproxyservice.integration.pu.v5.configuration.configuration.PuConstants.COORDINATION_NUMBER_MONTH_INDEX;
+import static se.inera.intyg.intygproxyservice.integration.pu.v5.configuration.configuration.PuConstants.COORDINATION_NUMBER_MONTH_VALUE_MIN;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+import se.inera.intyg.intygproxyservice.integration.api.pu.PuPersonsRequest;
+import se.inera.intyg.intygproxyservice.integration.api.pu.PuPersonsResponse;
+import se.inera.intyg.intygproxyservice.integration.api.pu.PuRequest;
+import se.inera.intyg.intygproxyservice.integration.api.pu.PuResponse;
+import se.riv.strategicresourcemanagement.persons.person.getpersonsforprofile.v5.rivtabp21.GetPersonsForProfileResponderInterface;
+import se.riv.strategicresourcemanagement.persons.person.getpersonsforprofileresponder.v5.GetPersonsForProfileType;
+import se.riv.strategicresourcemanagement.persons.person.v5.IIType;
+import se.riv.strategicresourcemanagement.persons.person.v5.LookupProfileType;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+@Profile(PU_PROFILE_V5)
+public class PuClientV5 {
+
+  private final GetPersonsForProfileResponderInterface getPersonsForProfileResponderInterface;
+
+  private final GetPersonsForProfileResponseTypeHandlerV5 getPersonsForProfileResponseTypeHandlerV5;
+
+  @Value("${integration.pu.logical.address}")
+  private String logicalAddress;
+
+  public PuResponse findPerson(PuRequest puRequest) {
```

Comment:
Maybe we should inclue the logging module and add performance logs to this api request? Just to track the time it takes to make PU-calls? What do you think?

Same for findPersons

Reply:
It's already added in the controller, but I can add it down here also if we want two separate times

Reply:
We tracked the time it takes to make external calls in other services, so thought it would be a good idea to do here aswell.